src/Sylius/Component/Core/Model/Order.php

```diff
@@ -491,7 +491,7 @@ public function getLastShipment()
     public function isInvoiceAvailable()
     {
         if (false !== $lastShipment = $this->getLastShipment()) {
-            return in_array($lastShipment->getState(), [ShipmentInterface::STATE_RETURNED, ShipmentInterface::STATE_SHIPPED]);
+            return in_array($lastShipment->getState(), [ShipmentInterface::STATE_SHIPPED]);
         }
 
         return false;
```

Comment:
Not sure if we should hardcode this logic on model, but if so I would make it a single line:
`return OrderShippingStates::STATE_SHIPPED === $this->shippingState`